app/metrics.yaml

```diff
@@ -7654,6 +7654,22 @@ awesomebar:
     metadata:
       tags:
         - Search
+  search_term_suggestion_clicked:
+    type: event
+    description: |
+      The search term suggestion in awesomebar was clicked.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/25816
+    data_reviews:
+      - ???????
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
```

Comment:
Is we're collecting this permanently, please add kbrosnan@mozilla.com as well as a contact.  Thanks